setup.cfg

```diff
@@ -29,14 +29,28 @@ classifiers =
     License :: OSI Approved :: MIT License
 
 [options]
+package_dir = =src
```

Comment:
```suggestion
package_dir = src
```

Reply:
This is the correct syntax; each value is a key-value pair separated by an =, and here the key is empty string.

Reply:
I totally missed this when reviewing https://setuptools.readthedocs.io/en/latest/setuptools.html :p
```
[options]
package_dir=
    =src
```